db/migrate/20231215172100_remove_foreign_keys_from_room_guests.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveForeignKeysFromRoomGuests < ActiveRecord::Migration[7.0]
+  def change
+    remove_reference :guests, :reservation, null: false, foreign_key: true
+    remove_reference :guests, :guest, null: false, foreign_key: true
+    remove_reference :guests, :room, null: false, foreign_key: true
```

Comment:
Nice work fixing forward! This is the best practice. but, in a timed setting like this it's also okay to run `rails db:drop` and then delete the migration file and start again.